docs/conf.py

```diff
@@ -63,70 +65,71 @@
 nitpicky = True
 nitpick_ignore = [
     # external library / undocumented external
-    ("py:class", "BaseModel"),
-    ("py:class", "pydantic.main.BaseModel"),
-    ("py:class", "pydantic.generics.GenericModel"),
-    ("py:class", "redis.asyncio.Redis"),
-    ("py:class", "sqlalchemy.orm.decl_api.DeclarativeMeta"),
-    ("py:class", "sqlalchemy.sql.sqltypes.TupleType"),
-    ("py:class", "sqlalchemy.dialects.postgresql.named_types.ENUM"),
-    ("py:class", "_orm.Mapper"),
-    ("py:class", "_orm.registry"),
-    ("py:class", "_schema.MetaData"),
-    ("py:class", "_schema.Table"),
-    ("py:class", "_RegistryType"),
-    ("py:class", "abc.Collection"),
-    ("py:class", "TypeEngine"),
-    ("py:class", "ExternalType"),
-    ("py:class", "UserDefinedType"),
-    ("py:class", "_types.TypeDecorator"),
+    (PY_CLASS, "BaseModel"),
```

Comment:
I understand why, but I don't think this really improves anything, as these directives aren't ever going to change.

If we are to go through with it,  `py:meth` and `py:obj` should be treated the same for sake of consistency.

Reply:
its about repetation of values - sonar doesnt like copy pasting.

Reply:
Yes, as I said, I understand why, but I don't think it adds anything. Not a blocker though, although I do think the consistency point is pertinent.

Reply:
addressed